components/resources/library/src/iosMain/kotlin/org/jetbrains/compose/resources/Resource.ios.kt

```diff
@@ -18,8 +18,10 @@ actual fun resource(path: String): Resource = UIKitResourceImpl(path)
 @ExperimentalResourceApi
 private class UIKitResourceImpl(path: String) : AbstractResourceImpl(path) {
     override suspend fun readBytes(): ByteArray {
-        val absolutePath = NSBundle.mainBundle.resourcePath + "/" + path
```

Comment:
We have a demo project https://github.com/JetBrains/compose-multiplatform/tree/master/components/resources/demo that is used to test the functionality. It should be synchronized with the changes in the library. So we need to migrate iOS part of the demo to new functionality for this and test it with locally build. When a dev build is available (after merge of this PR) it is better to update compose version to a dev build of the project.